requirements.txt

```diff
@@ -4,17 +4,18 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-amqp==1.4.9               # via kombu
+-e git+https://github.com/steve-gregory/billiard.git#egg=billiard
```

Comment:
Basically, we need to have a stable copy of all related repositories so that we have this project under source control.  If we don't, we will very quickly have an unstable version.